situation_room/mix.exs

```diff
@@ -0,0 +1,21 @@
+defmodule SituationRoom.MixProject do
+  use Mix.Project
+
+  def project do
+    [
+      apps_path: "apps",
```

Comment:
What was the thought process/reasoning for making this an umbrella application vs just a single application?

Reply:
This was my mistake for assuming future features again. I will remove the umbrella app and make this just a single app.